src/pages/cli/usage/command-hooks.mdx

```diff
@@ -0,0 +1,250 @@
+export const meta = {
+  title: `Command Hooks`,
+  description: `Execute custom scripts before, during, and after Amplify CLI commands (“amplify push”, “amplify api gql-compile”, and more). This allows you to extend Amplify’s best-practice defaults to meet your organization’s specific security guidelines and operational requirements.`,
```

Comment:
This seems a little verbose for the page description. I'd suggest just the first sentence here

Reply:
Keeping as is for SEO purposes. 
```suggestion
  description: `Execute custom scripts before, during, and after Amplify CLI commands (“amplify push”, “amplify api gql-compile”, and more).`,
```